src/utils/parseIP.ts

```diff
@@ -0,0 +1,4 @@
+export function parseIP(ip: string) {
```

Comment:
why have you created a new utils file for this function and not for the other ones?